packages/wallets/torus/package.json

```diff
@@ -31,7 +31,8 @@
         "@solana/web3.js": "^1.20.0",
         "@toruslabs/solana-embed": "^0.1.0",
         "assert": "npm:assert@^2.0.0",
-        "stream": "npm:stream-browserify@^3.0.0"
+        "stream": "npm:stream-browserify@^3.0.0",
+        "stream-browserify": "^3.0.0"
```

Comment:
There does exist the possibility of linking one to the other using `"stream": "link:./node_modules/stream-browserify"` instead of hardcoding the dependency twice, but that would make this `package.json` compatible with Yarn _only_.